Tasks/CMakeV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,10 +3,13 @@
   "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613719)",
   "loc.description": "Build with the CMake cross-platform build system",
   "loc.instanceNameFormat": "CMake $(cmakeArgs)",
+  "loc.group.displayName.advanced": "Advanced",
   "loc.input.label.cwd": "Working Directory",
   "loc.input.help.cwd": "Current working directory when cmake is run.",
   "loc.input.label.cmakeArgs": "Arguments",
   "loc.input.help.cmakeArgs": "Arguments passed to cmake",
+  "loc.input.label.runInsideShell": "Run cmake command inside shell",
+  "loc.input.help.runInsideShell": "CMake arguments will be handled like they would be inside of an OS specific shell. It can be used to handle environment variables in argument string.",
```

Comment:
Suggestion: "It can be used to handle environment variables inside of argument strings."

Reply:
@leantk Thank you!